### PR TITLE
Add release notes for 1.27.3

### PR DESCRIPTION
**Pull request by ikhoon**

### CODE REVIEW

File: site/src/pages/release-notes/1.27.3.mdx [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+date: 2024-03-26
+---
+
+## 🛠️ Bug fixes
+
+- You no longer see `NullPointerException` when creating Thrift clients in a multi-threaded environment. #5497
+- `IllegalReferenceCountException` no longer occurs during WebSocket handshake. #5495
+- When a client sends a `CLOSE` frame, the user provided subscription is correctly closed #5482
```
Comment (minwoox) [inline site/src/pages/release-notes/1.27.3.mdx:9]:
```suggestion
- When a client sends a `CLOSE` frame, the user provided subscription is correctly closed. #5482
```

**Review by minwoox — APPROVED** [review]
Thanks!

**Comment by ikhoon** [comment]
image

**Review by jrhee17 — APPROVED** [review]

**Comment by ikhoon** [comment]
Thanks for the quick review. 🙇‍♂️

**Pull request merged**